holder.yml

```diff
@@ -56,6 +56,39 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /exchanges/:
+    get:
+      summary: provides a discovery endpoint for the exchanges supported by this server endpoint.
+      operationId: discoverExchanges
+      description: 
+        This endpoint returns a map of the exchange-ids (endpoints) supported by this server, 
+        and the associated protocol supported by that exchange endpoint in an 
+        "exchange-id" -> "protocol" map. The keys in the returned object MUST be a valid 
+        ExchangeId path parameter, and the value field MUST be a string that 
+        references the supported protocol on that endpoint. Protocol strings MUST be 
+        the protocol specification name or type, or a URL to a spec. Clients consuming this 
+        list wishing to use an exchange endpoint MUST recognize and support the protocol identified
+        in the value field. Clients are not expected to dynamically process the protocol specified.    
+      responses:
+        "200":
+          description: a map of the exchange-id endpoints to protocols those exchanges support.
+          content:
+            application/json:
+              schema:
+                type: object 
+                additionalProperties: 
+                  type: string
```

Comment:
don't return a map... return an array with the potential to support paging, etc.